localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -99,7 +99,8 @@ NDTScanMatcher::NDTScanMatcher()
   regularization_enabled_(declare_parameter("regularization_enabled", false)),
   estimate_scores_for_degrounded_scan_(
     declare_parameter("estimate_scores_for_degrounded_scan", false)),
-  z_margin_for_ground_removal_(declare_parameter("z_margin_for_ground_removal", 0.8))
+  z_margin_for_ground_removal_(declare_parameter("z_margin_for_ground_removal", 0.8)),
+  critical_upper_bound_exe_time_ms_(24)
```

Comment:
@Motsu-san 
I apologize for interrupting the conversation.
What is the specific `affect` in this comment?
> I would like to avoid settings that affect the applications this warning is followed by.

Also, defining default value in this case, 100 you proposed is ok but if you don't define in the parameter file in [autoware_launch](https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/config/localization/ndt_scan_matcher.param.yaml), the parameter can't be changed from yaml file.

Share the link about [autoware_launch](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/launch-files/#autoware_launch) and [paramter decleration](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/#declare-parameter-function) for your information.

This is also FYI,
In planning/control module, remove all the default value in this [issue](https://github.com/autowarefoundation/autoware.universe/issues/2969) to avoid situation forgetting defining parameter in autoware_launch.

Reply:
@kyoichi-sugahara 
Thank you for your comment.

> What is the specific affect in this comment?

You mean, "You would like to know what application I said above."
Currently, there is a few on Leo Drive, I think. But in the future, more people will use this warning. In that case, they would wonder why the value is 24ms and how they adjust this value to thier purpose.
100ms is needed for NDT's default cycle. so I showed one example easy to avoid side-effects.

Reply:
@Motsu-san 
I wandered what is your concern. 
I just couldn't understand and now it's clear thanks!